### DISCUSSION TITLE
Focusing grating couplers

### DISCUSSION THREAD

**Discussion by gbfarias** [HelgeGehring/gdshelpers#35] [Q&A]
A question regarding the grating couplers - do the grating lines follow the typical equations for focusing grating couplers (as in Hochberg's book - Silicon photonics design: from devices to systems) or are they just circular shapes?

**Comment by HelgeGehring** [comment]
At the moment the constructur of `GratingCoupler` takes only a list of radii as an argument, so it's only circular shapes. But it should be straight forward to extend it to take two values instead of one to also allow elliptical shapes.
If I see it right, that should be sufficient get the coupler-shape from the book(?)
